src/romein.cu

```diff
@@ -182,17 +209,13 @@ inline void launch_romein_kernel(int      nbaseline,
     cudaDeviceProp dev;
     cudaError_t error;
     error = cudaGetDeviceProperties(&dev, 0);
-     if(error != cudaSuccess)
-     {
-        printf("Error: %s\n", cudaGetErrorString(error));
-      }
-
+    if(error != cudaSuccess) printf("Error: %s\n", cudaGetErrorString(error));
 	
     int blk_cnt ;
     int block_x=maxsupport*maxsupport ;
     int tile_y ;
-    if(block_x==1)tile_y=nbaseline;
-    else tile_y = dev.maxThreadsPerBlock/block_x;
+    if(block_x==1)tile_y=std::min(nbaseline,dev.maxThreadsPerBlock);
+    else tile_y = std::min((int)nbaseline/block_x,dev.maxThreadsPerBlock);
```

Comment:
I thought the code was going to move away from `dev.maxThreadsPerBlock` to better match the Bifrost style.